lib/r10k/git/cache.rb

```diff
@@ -16,7 +16,7 @@ class R10K::Git::Cache
 
   include R10K::Settings::Mixin
 
-  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/git': '/root/.r10k/git')
+  def_setting_attr :cache_root, File.expand_path(ENV['HOME'] ? '~/.r10k/cache': '/root/.r10k/cache')
```

Comment:
Yes and no. In r10k documentation, we don't talk about the cache in a way which suggests the user might do anything with it. It might be worth noting this in the CHANGELOG, for those interested in that level of detail? I don't have another location which seems appropriate for a mid-version cache clearing.